pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2318,4 +2318,9 @@ self: super: {
   # Too strict bounds on chell: https://github.com/fpco/haskell-filesystem/issues/24
   system-fileio = doJailbreak super.system-fileio;
 
+  # 2022-02-28: Unmaintained. Distribution.Simple must have changed since 2016
+  digits = appendPatches [
+    ./patches/digits-fix-162266.patch
+  ] super.digits;
```

Comment:
Inspiration for a simpler fix can be taken from `csv` and we could do something like:

```
  # Upstream ships a broken Setup.hs
  digits = overrideCabal (drv: { prePatch = "rm Setup.hs"; }) super.digits;
```

If `Setup.hs` is missing, `generic-builder.nix` creates a default one which should be enough to build the package here. This would also allow us to drop the patch.